src/invidious/frontend/comments_youtube.cr

```diff
@@ -149,12 +149,12 @@ module Invidious::Frontend::Comments
 
         if comments["videoId"]?
           html << <<-END_HTML
-            <a href="https://www.youtube.com/watch?v=#{comments["videoId"]}&lc=#{child["commentId"]}" title="#{translate(locale, "YouTube comment permalink")}">[YT]</a>
+            <a rel="noreferrer noopener" href="#{YT_URL.to_s}/watch?v=#{comments["videoId"]}&lc=#{child["commentId"]}" title="#{translate(locale, "YouTube comment permalink")}">[YT]</a>
```

Comment:
Can you please revert the `YT_URL` change? Youtube is unlikely to change their primary domain soon, and I wanted to get rid og that useless `YT_URL` constant!